backend/service_accounts/templates/admin/service_accounts/serviceaccount/issue_token.html

```
@@ -0,0 +1,93 @@
+            <div class="form-row">
+                <div>
+                    <label>Scopes:</label>
+                    <div class="readonly">{{ token.scopes|join:", " }}</div>
+                </div>
+            </div>
+            <div class="form-row">
+                <div>
+                    <label>Plaintext token:</label>
+                    <div class="readonly">
+                        <input
+                            type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)